scripts/clang_format

```diff
@@ -0,0 +1 @@
+../python/mpi_cmake_modules/clang_format.py
```

Comment:
I assume this serves as a kind of alias for `clang_format.py`.  Does it work from everywhere when using a relative path?  I think a symlink would be a bit cleaner.  Or simply remove the `.py` from the filename (this is a common thing for ROS nodes implemented in Python).

Reply:
This is a symlink

Reply:
Oh, then forget what I said.  In the diff it looks like it is just a text file with the path written in it.

Reply:
Yes because symlink are basically that^^